ros/api/v1/hosts.py

```diff
@@ -80,6 +80,16 @@ class HostsApi(Resource):
         'max_io': fields.Float,
         'io_all': fields.Raw
     }
+
+    groups_fields = {
```

Comment:
Could you also include the changes to [openapi spec](https://github.com/RedHatInsights/ros-backend/blob/main/ros/api/v1/openapi_spec.py)

Reply:
sure @r14chandra , will push the changes in the latest commit.